account_payment_order/models/account_payment_order.py

```diff
@@ -330,7 +330,7 @@ def generated2uploaded(self):
         return True
 
     @api.multi
-    def _prepare_move(self):
+    def _prepare_move(self, bank_lines=None):
```

Comment:
Then is the purpose to change bank payment lines? Then that should be done in another method I think